adaptors_task/student/src/impl.cpp

```diff
@@ -1,48 +1,58 @@
 #include "impl.h"
 #include <vector>
 #include <stack>
-#include <algorithm>
 
 const std::vector<std::pair<char, char>> brackets {
     {'(', ')'},
     {'[', ']'},
     {'{', '}'}
 };
 
+bool isBracket(char c, char& matchOpenBracket, bool& isOpening, bool& isClosing)
+{
+    for (auto& bracket : brackets)
+    {
+        if (bracket.first != c && bracket.second != c)
```

Comment:
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es77-minimize-the-use-of-break-and-continue-in-loops